src/s1reader/s1_reader.py

```diff
@@ -19,17 +19,21 @@
 from nisar.workflows.stage_dem import check_dateline
 
 from s1reader import s1_annotation  # to access __file__
-from s1reader.s1_annotation import RFI_INFO_AVAILABLE_FROM, ProductAnnotation, NoiseAnnotation,\
-                                   CalibrationAnnotation, AuxCal,\
-                                   BurstCalibration, BurstEAP, BurstNoise,\
-                                   BurstExtendedCoeffs, SwathRfiInfo, SwathMiscMetadata
+from s1reader.s1_annotation import (RFI_INFO_AVAILABLE_FROM, ProductAnnotation,
+                                    NoiseAnnotation, CalibrationAnnotation,
+                                    AuxCal, BurstCalibration,
+                                    BurstEAP, BurstNoise,
+                                    BurstExtendedCoeffs, SwathRfiInfo,
+                                    SwathMiscMetadata)
 
 from s1reader.s1_burst_slc import Doppler, Sentinel1BurstSlc
 from s1reader.s1_burst_id import S1BurstId
-from s1reader.s1_orbit import T_ORBIT
+from s1reader.s1_orbit import (T_ORBIT, merge_osv_list)
+from s1reader.s1_orbit import FMT as orbitfile_datetime_format
 
 # Tolerance of the ascending node crossing time in seconds
 ASCENDING_NODE_TIME_TOLERANCE_IN_SEC = 1.0
+PADDING_TIME_ORBIT = 60.0
```

Comment:
Is this padding meant to be the same as `PADDING_SHORT` in `s1_orbit.py`? If so, let's move it there so there's a single parameter instead two that do the same.

Reply:
`s1_orbit.py` has a constant `PADDING_SHORT` that does the similar thing. I've changed this code to import the constant from `s1_orbit.py`